res/TensorFlowLiteRecipes/Net_Conv_Add_Mul_004/test.qconf.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
To express this network for quantization test, how about using name
`Quant_Conv_Add_Mul_001`?